Modules/rat_rescue.py

```diff
@@ -83,7 +94,7 @@ def __init__(self, case_id: UUID, client: str, system: str, irc_nickname: str,
         self._epic: bool = epic
         self._codeRed: bool = code_red
         self._outcome: None = None
-        self._title: str = title
+        self._title: Union(str, None) = title
```

Comment:
This might just be a typo, but I believe `Union[str, None]` is what you need here like you have it in the signature. Not actually sure what `Union(str, None)` would do.

Reply:
That was a typo i thought i fixed. but yes, `()` will cause unintended behavior (raise an exception) which is why i had to fix it in 6aefa192.